tests/chainer_tests/functions_tests/loss_tests/test_huber_loss.py

```diff
@@ -96,8 +96,15 @@ def test_backward_gpu(self):
 
     def check_double_backward(self, x_data, t_data, y_grad, x_grad_grad,
                               t_grad_grad):
+
+        delta = 1
+        eps = self.double_backward_options['eps']
+        mask = numpy.abs(numpy.abs(x_data - t_data) - delta) < eps
```

Comment:
Could you support gpu tests (with old numpy)?

Reply:
Fixed.